chrome/fed-modules.json

```
@@ -440,9 +440,34 @@
             }
         ]
     },
+    "rhosak": {
+        "manifestLocation": "/apps/rhosak/fed-mods.json",
+        "analytics": {
+            "APIKey": "eGAB8pv9DFgdj80STEaeHSMvRPAjdDUH"
+        },
+        "modules": [
+            {
+                "id": "rhosak",
+                "module": "./RootApp",
+                "routes": ["/application-services/streams"]
+            }
+        ]
+    },
+    "srs": {
+        "manifestLocation": "/beta/apps/srs-ui-build/fed-mods.json",
```

No need to include the `/beta`

```suggestion
        "manifestLocation": "/apps/srs-ui-build/fed-mods.json",
```

---

Thank you, done!